demo/dataset/README-CN.md

```diff
@@ -0,0 +1,99 @@
+# 数据预处理与准备
+在这个项目中，我们会统一所有Demo项目的数据处理过程，包括对MovieLens-1M，MovieLens-25M，Criteo-5D等，以及其他等数据集。处理过程包括：特征生成、召回样本生成、排序样本生成、负采样等具体等工作。
+
+## 初始化模型配置文件
+首先，我们需要初始化配置文件，我们需要通过给出的 YAML 配置模版对不同阶段的配置文件进行初始化，主要是替换模版中一些需要定制的变量。举例来说，我们需要替换自己具体的 S3 路径 `MY_S3_BUCKET`:
+
+```shell
+export MY_S3_BUCKET='your S3 bucket directory'
+envsubst < fg.yaml > fg.yaml.dev 
+```
+
+在后面我们运行 python 脚本之前，我们假设大家已经完成了初始化模型配置文件的工作。如果还没有安装 `envsubst` 命令，可以进行安装，比如在基于 `Debian` 的 Linux 系统上，可以使用 `sudo apt-get install gettext-base` 命令来完成安装。
+
+## MovieLens-1M
+在这一节，我们使用 [MoiveLens-1M](https://grouplens.org/datasets/movielens/1m/) 来进行演示。您可以从上面提供的网址中下载项目所需要的数据，并存储在您的云端 S3 存储上。
+
+### 1. 特征生成
+假设我们位于dataset项目的根目录，我们可以通过执行以下命令来完成特征生成的工作：
+
+ ```shell
+ cd ml_1m
+ python fg.py --conf fg.yaml.dev --verbose
+ ```
+
+### 2. 召回样本生成
+假设我们位于dataset项目的根目录，我们可以通过执行以下命令准备好 `Collaborative Filtering` 算法训练所使用的样本数据：
```

Comment:
It's better give a link to the corresponding algo/model(CF/TwoTowers/CTR) training code. Users can better relate the dataset processing with model training.

Reply:
I fix that for CTR models now.